portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -56,127 +64,232 @@ public void tearDown() {
     helper.tearDown();
   }
 
-  @Test 
-  public void testPOSTSingleComment() throws Exception {
-    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
-    HttpServletRequest postRequest = mock(HttpServletRequest.class);
-    HttpServletResponse postResponse = mock(HttpServletResponse.class);
+  private String[] convertStringToArray(String responseOutput) {
+      //Gets rid of brackets
```

Comment:
style nit - add a space between the `//` ane the comment (and below, as well)

Reply:
done!